ansible_collections/arista/avd/roles/eos_validate_state/README.md

```diff
@@ -85,6 +85,18 @@ root_dir: '{{ inventory_dir }}'
 eos_validate_state_name: 'reports'
 eos_validate_state_dir: '{{ root_dir }}/{{ eos_validate_state_name }}'
 
+# Tests to include
```

Comment:
As it is variable description, we should put:

```
eos_validate_state_bgp_check: < true | false (Default: true) >
...
```

Reply:
OK I will fix this